src/XliffTasks/buildCrossTargeting/XliffTasks.targets

```diff
@@ -28,4 +28,42 @@
              />
   </Target>
 
+  <!--
+    Sorts the trans-unit elements in the .xlf files by the value of their id attributes.
+    Runs a sort per target framework as resource files may be conditioned on the framework.
+  -->
+  <Target Name="SortXlf">
+    <ItemGroup>
+      <_XlfTargetFramework Include="$(TargetFrameworks)" />
+    </ItemGroup>
+
+    <!--
+      These inner builds cannot be parallelized as they would race to 
+      update the same inner xlf files.
+    -->
+    <MSBuild Projects="$(MSBuildProjectFile)"
+             Targets="SortXlf"
+             Properties="TargetFramework=%(_XlfTargetFramework.Identity)"
+             />
+  </Target>
+
+  <!--
+    Checks that all trans-unit elements in the .xlf files have an up-to-date translation.
+    Runs a sort per target framework as resource files may be conditioned on the framework.
+  -->
+  <Target Name="EnsureAllResourcesTranslated">
+    <ItemGroup>
+      <_XlfTargetFramework Include="$(TargetFrameworks)" />
+    </ItemGroup>
```

Comment:
I believe this will cause redundant inner builds if more than one of these is invoked in the same build: `msbuild /t:UpdateXlf;SortXlf;EnsureAllResourcesAreTranslated`